core/field_angle.js

```diff
@@ -272,22 +272,18 @@ Blockly.FieldAngle.prototype.render_ = function() {
  *     or undefined if triggered programmatically.
  * @protected
  */
-Blockly.FieldAngle.prototype.showEditor_ = function(opt_e) {
+FieldAngle.prototype.showEditor_ = function(opt_e) {
   // Mobile browsers have issues with in-line textareas (focus & keyboards).
-  var noFocus =
-      Blockly.utils.userAgent.MOBILE ||
-      Blockly.utils.userAgent.ANDROID ||
-      Blockly.utils.userAgent.IPAD;
-  Blockly.FieldAngle.superClass_.showEditor_.call(this, opt_e, noFocus);
+  var noFocus = userAgent.MOBILE || userAgent.ANDROID || userAgent.IPAD;
```

Comment:
Here and in several other places later on, we should be updating `var` to `let`/`const`